open_dread_rando/files/levels/s080_shipyard.lc.lua

```diff
@@ -473,6 +473,7 @@ function s080_shipyard.OnSubAreaChange(old_subarea, old_actorgroup, new_subarea,
   end
 
   Scenario.UpdateProgressiveItemModels()
+  Scenario.UpdateRoomName("Hanubia", new_subarea)
```

Comment:
since we've got multiple things happening at the end of each of these functions now, we should wrap them into a single function in our custom scenario file, and pass all 5 args to it. instead of using a hardcoded readable scenario name here, we can just use the internal scenario ID for everything, since we can grab the current scenario id at any time